pydov/search/interpretaties.py

```diff
@@ -745,3 +746,126 @@ def search(self, location=None, query=None, return_fields=None):
             columns=GeotechnischeCodering.get_field_names(
                 return_fields))
         return df
+
+
+class QuartaireStratigrafieSearch(AbstractSearch):
+    """Search class to retrieve the interpretation for Quartair
+    stratigrafie"""
+
+    __wfs_schema = None
+    __wfs_namespace = None
+    __md_metadata = None
+    __fc_featurecatalogue = None
+
+    def __init__(self):
+        """Initialisation."""
+        super(QuartaireStratigrafieSearch, self).__init__(
+            'interpretaties:quartaire_stratigrafie', QuartaireStratigrafie)
```

Comment:
The WFS service needs to remain for things to keep working.

```suggestion
            'interpretaties:quartaire_stratigrafie', QuartairStratigrafie)
```